run_length_encoding/rebar.config

```diff
@@ -0,0 +1,2 @@
+{erl_opts, [debug_info]}.
+{deps, []}.
```

Comment:
configure your editor to always add a newline at the end of the file. http://adaptivepatchwork.com/2012/03/01/mind-the-end-of-your-line/